api/api/settings.py

```diff
@@ -97,6 +97,21 @@
     'EXCEPTION_HANDLER': 'desecapi.exception_handlers.exception_handler',
     'DEFAULT_VERSIONING_CLASS': 'rest_framework.versioning.NamespaceVersioning',
     'ALLOWED_VERSIONS': ['v1', 'v2'],
+    'DEFAULT_THROTTLE_CLASSES': [
+        'desecapi.throttling.ScopedRatesThrottle',
+        'rest_framework.throttling.UserRateThrottle',
+    ],
+    'DEFAULT_THROTTLE_RATES': {
+        # ScopedRatesThrottle
+        'account_management_active': ['3/min'],  # things that send email or create domains on pdns
```

Comment:
creating domains on pdns should go into the `dns_api_write` group, no?

Reply:
Well, yes. What's meant here is creating a dyn domain when clicking the account activation link. This can happen with the same rate as the activation email can be sent. Does it make sense? Maybe I need to improve the comment?

Reply:
yup let's make clear here "create" here only refers to creation of domains as part of user registration process

Reply:
fixed comment